### ISSUE TITLE
File not found error on startup

### ISSUE THREAD

**Issue by Xyndra** [FoundryMC/Veil#20]
```
[21:23:25] [Worker-Main-7/ERROR] (Veil) Error loading resource: minecraft:datapacks/bundle/pack.mcmeta
java.io.FileNotFoundException: minecraft:datapacks/bundle/pack.mcmeta
        at net.minecraft.server.packs.resources.ResourceProvider.method_43043(ResourceProvider.java:21) ~[minecraft-merged-b313045ff1-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
        at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
        at net.minecraft.server.packs.resources.ResourceProvider.getResourceOrThrow(ResourceProvider.java:21) ~[minecraft-merged-b313045ff1-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
        at net.minecraft.server.packs.resources.ResourceProvider.open(ResourceProvider.java:25) ~[minecraft-merged-b313045ff1-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
        at foundry.veil.impl.resource.loader.McMetaResourceLoader.load(McMetaResourceLoader.java:34) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.visitResource(VeilResourceManagerImpl.java:118) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.lambda$loadPack$0(VeilResourceManagerImpl.java:79) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at net.minecraft.server.packs.VanillaPackResources$Anonymous$41e828eb6aae434392456727bec13f37.visitFile(VanillaPackResourcesMixin.java:67) ~[?:?]
        at net.minecraft.server.packs.VanillaPackResources$Anonymous$41e828eb6aae434392456727bec13f37.visitFile(VanillaPackResourcesMixin.java:48) ~[?:?]
        at java.nio.file.Files.walkFileTree(Files.java:2811) ~[?:?]
        at java.nio.file.Files.walkFileTree(Files.java:2882) ~[?:?]
        at net.minecraft.server.packs.VanillaPackResources.veil$listResources(VanillaPackResources.java:548) ~[minecraft-merged-b313045ff1-1.20.1-loom.mappings.1_20_1.layered+hash.2198-v2.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.loadPack(VeilResourceManagerImpl.java:77) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.lambda$reloadClient$4(VeilResourceManagerImpl.java:156) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
        at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276) ~[?:?]
        at java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.lambda$reloadClient$5(VeilResourceManagerImpl.java:154) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
[21:23:25] [Worker-Main-7/WARN] (Veil) findResources in namespace veil-example-mod, mod veil-example-mod failed!
java.nio.file.NoSuchFileException: D:\Code\Minecraft\veil-example-mod\build\classes\java\main\assets\veil-example-mod
        at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:85) ~[?:?]
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:103) ~[?:?]
        at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:108) ~[?:?]
        at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:53) ~[?:?]
        at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:38) ~[?:?]
        at sun.nio.fs.WindowsFileSystemProvider.readAttributes(WindowsFileSystemProvider.java:199) ~[?:?]
        at java.nio.file.Files.readAttributes(Files.java:1851) ~[?:?]
        at java.nio.file.FileTreeWalker.getAttributes(FileTreeWalker.java:220) ~[?:?]
        at java.nio.file.FileTreeWalker.visit(FileTreeWalker.java:277) ~[?:?]
        at java.nio.file.FileTreeWalker.walk(FileTreeWalker.java:323) ~[?:?]
        at java.nio.file.Files.walkFileTree(Files.java:2804) ~[?:?]
        at java.nio.file.Files.walkFileTree(Files.java:2882) ~[?:?]
        at net.fabricmc.fabric.impl.resource.loader.ModNioResourcePack.veil$listResources(ModNioResourcePack.java:560) ~[fabric-resource-loader-v0-0.11.10+1802ada577.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.loadPack(VeilResourceManagerImpl.java:77) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.lambda$reloadClient$4(VeilResourceManagerImpl.java:156) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183) ~[?:?]
        at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276) ~[?:?]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
        at java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276) ~[?:?]
        at java.util.AbstractList$RandomAccessSpliterator.forEachRemaining(AbstractList.java:720) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150) ~[?:?]
        at java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596) ~[?:?]
        at foundry.veil.impl.resource.VeilResourceManagerImpl.lambda$reloadClient$5(VeilResourceManagerImpl.java:154) ~[Veil-fabric-1.20.1-1.0.0.254.jar:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.run(CompletableFuture.java:1768) ~[?:?]
        at java.util.concurrent.CompletableFuture$AsyncSupply.exec(CompletableFuture.java:1760) ~[?:?]
        at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373) ~[?:?]
        at java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182) ~[?:?]
        at java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655) ~[?:?]
        at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622) ~[?:?]
        at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165) ~[?:?]
```

**Comment by lars2800** [comment]
I always get that warning but i just ignore it

I think java.io.FileNotFoundException: minecraft:datapacks/bundle/pack.mcmeta is not important and that it is just somthing from minecraft itself not the mod

This one: java.nio.file.NoSuchFileException: D:\Code\Minecraft\veil-example-mod\build\classes\java\main\assets\veil-example-mod is a little weird cause i think that build is an temporary folder, i guess this is just some misconfiguration in the base template or somthing **i don't know java very well**.

**Comment by Ocelot5836** [comment]
Fixed in d38479ce
